soap/pom.xml

```diff
@@ -0,0 +1,88 @@
+<!--
```

Comment:
Add `<?xml version="1.0" encoding="UTF-8"?>` to the first line of the file and build will pass

Reply:
And here it is :)